Portfolio/Portfolio.html

```diff
@@ -9,7 +9,8 @@
   <body>
     <div class="informationContainer">
       <div class="loginSection">
-        <button class="loginText" id="loginButton" onclick="redirectToSignupPage()">Login</button>
+        <button class="loginText" onclick="redirectToEditPage()">Edit</button>
+        <button class="loginText" onclick="redirectToSignupPage()">Signup</button>
```

Comment:
use consistent programming patterns/styles. In one file you are using javascript event handlers. Here you are using html inline events. Use one approach consistently throught your codebase.

Reply:
Fixed, Thank you